onnx2torch/node_converters/roialign.py

```diff
@@ -31,23 +33,57 @@ def __init__(
         self._sampling_ratio = sampling_ratio
         self._spatial_scale = spatial_scale
 
-    def forward(
-            self,
+    @staticmethod
+    def _do_forward(
             input_tensor: torch.Tensor,
             rois: torch.Tensor,
             batch_indices: torch.Tensor,
+            output_size: Tuple[int, int],
+            spatial_scale: float,
+            sampling_ratio: int,
     ) -> torch.Tensor:
-        batched_rois = torch.concat([batch_indices.unsqueeze(1).to(rois.dtype), rois], dim=1)
+
+        batch_indices = batch_indices.unsqueeze(1).to(rois.dtype)
+
+        batched_rois = torch.concat([batch_indices, rois], dim=1)
 
         return roi_align(
             input=input_tensor,
             boxes=batched_rois,
-            output_size=self._output_size,
-            spatial_scale=self._spatial_scale,
-            sampling_ratio=self._sampling_ratio,
+            output_size=output_size,
+            spatial_scale=spatial_scale,
+            sampling_ratio=sampling_ratio,
             aligned=False,
         )
 
+    def forward(
+            self,
+            input_tensor: torch.Tensor,
+            rois: torch.Tensor,
+            batch_indices: torch.Tensor,
+
+    ) -> torch.Tensor:
+
+        output = self._do_forward(
+            input_tensor,
+            rois,
+            batch_indices,
+            self._output_size,
+            self._spatial_scale,
+            self._sampling_ratio,
+        )
+        if torch.onnx.is_in_onnx_export():
+            args = [input_tensor, rois, batch_indices]
+            return _RoiAlignExportToOnnx.set_output_and_apply(output, *args)
+
+        return output
+
+class _RoiAlignExportToOnnx(CustomExportToOnnx):  # pylint: disable=abstract-method
+
+    @staticmethod
+    def symbolic(graph: torch_C.Graph, *args) -> torch_C.Value:
+        return graph.op('RoiAlign', *args, outputs=1)
```

Comment:
You forgot about attributes
```python
output_height : int (default is 1)
output_width : int (default is 1)
sampling_ratio : int (default is 0)
spatial_scale : float (default is 1.0)
```

Reply:
Done

Reply:
PEP 8
Immediately before a comma, semicolon, or colon:
# Correct:
if x == 4: print(x, y); x, y = y, x
# Wrong:
if x == 4 : print(x , y) ; x , y = y , x

Reply:
Done